src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -31,12 +32,14 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")
```

Comment:
In your solution, USER is the default role for each registered user. It means that ADMIN also has a USER role. So validating the USER role is enough for such endpoints